src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
If you're not using css from this file. Then delete this file as It is useless in that scenario.